include/Smilies.php

```diff
@@ -0,0 +1,182 @@
+<?php
+
+/**
+ * @file include/Smilies.php
+ * @brief This file contains the Smilies class which contains functions to handle smiles
+ */
+
+/**
+ * This class contains functions to handle smiles
```

Comment:
There are several addon which add new emojies or add a selection button to the editor, do these still work with the new class or will they need adoption?